docs/operate.md

```diff
@@ -2,11 +2,48 @@
 
 Information about operating the hubs, debugging problems, and performing common actions.
 
+(operate:team-process)=
+## Team process for operating hubs
+
+### Each hub gets an issue
+
+When a new hub is created, [create a new GitHub issue for the hub](https://github.com/2i2c-org/pilot-hubs/issues/new) using the "New Hub" template.
+This issue is a "meta" issue that tracks the state of this hub over time.
+These issues serve both as a "Source of Truth" of information about the hub, as well as a place for conversation around operating that hub.
+
+### Add the new hub issue to the hubs project
```

Comment:
Can we just use the issues to begin with and see how it goes?

Reply:
Yep that works for me 👍